ports/espressif/mpconfigport.mk

```diff
@@ -136,6 +135,7 @@ CIRCUITPY_BLEIO ?= 0
 else
 CIRCUITPY_BLEIO ?= 1
 endif
+CIRCUITPY_AUDIOMP3 ?= 1
```

Comment:
Why S3 only? I'd probably only enable it on boards with more than 4mb flash. There others are tight on space and this is a new feature.

Reply:
I only tested on S3. Do you want me to test on other boards, or just enable widely?

Reply:
I think you should try it everywhere. We do have size issues though.

Reply:
OK, the current build should enable it on anything with 4MB+